src/libopensc/card-cardos.c

```diff
@@ -159,49 +190,110 @@ static int cardos_have_2048bit_package(sc_card_t *card)
 	return 0;
 }
 
+
```

Comment:
clang-format suggested the following change:


```
-
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->